File: src/AHF-API/DataLayer/Model/Game.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DataLayer.Model
{
    public class Game
    {
        public Game()
        {
            Teams = new List<Team>();
        }
        public int Id { get; set; }
        public List<Team> Teams { get; set; }
        public List<Location> Locations { get; set; }
    }
}

File: src/AHF-App/app/src/main/java/com/ahf/antwerphasfallen/Model/Game.java
package com.ahf.antwerphasfallen.Model;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by Jorren on 20/10/2018.
 */

public class Game {
    private int id;
    private List<Team> teams;

    public Game() {
        teams = new ArrayList<>();
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public List<Team> getTeams() {
        return teams;
    }

    public void setTeams(List<Team> teams) {
        this.teams = teams;
    }
}
